tests/Parser/SalesPricesTest.php

```diff
@@ -17,12 +17,27 @@ public function parseProvider()
                 array(
                     'entries' => array(
                         array(
-                            'id' => '1',
+                            'id' => 3,
```

Comment:
- [ ] Move from `array` to `[ ]`

- [ ] Remove the comments and add them as descriptions directly to the data sets.